storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx

```diff
@@ -135,7 +135,7 @@ export default {
     },
     onCapture: {
       control: "events",
-      type: { required: true },
+      // type: { required: true },
```

Comment:
can we remove commented parameter

Reply:
resolved in this [ca7c463](https://github.com/mosip/mosip-plugins/pull/56/commits/ca7c463b799a7e2584b2318c9bd15a2fa66880e7)